src/apps/activities/tests/test_reusable_items.py

```diff
@@ -0,0 +1,75 @@
+from apps.shared.test import BaseTest
+from infrastructure.database import transaction
+
+
+class TestReusableItem(BaseTest):
+    fixtures = ["users/fixtures/users.json"]
+
+    login_url = "/auth/token"
+    create_url = "activity/item_choices"
+    update_url = "activity/item_choices"
+    delete_url = "activity/item_choices/{id}"
+
```

Comment:
First of all, we use starlette library via FastAPI. It provides catching the URL from the callback function (like names for URLs in Django).

Check this: https://github.com/encode/starlette/blob/c566fc6c819f0d565f8cff432351fe009e83d866/starlette/routing.py#L206

Reply:
https://stackoverflow.com/questions/63682956/fastapi-retrieve-url-from-view-name-route-name